docs/authorization/globus_app/config.rst

```diff
@@ -0,0 +1,33 @@
+.. _globus_app_config:
+
+.. currentmodule:: globus_sdk.globus_app
+
+GlobusApp Configuration
+=======================
+
+Reference
+---------
+
+Data Model
+^^^^^^^^^^
+
+.. autoclass:: globus_sdk.GlobusAppConfig()
+    :members:
+    :exclude-members: token_validation_error_handler
```

Comment:
I'm surprised by the decision to exclude this field from the documentation about the config. Can you share the reasoning behind this?

If we wanted it to be internal, we could have named it to be private. So I'm not clear on why it would be a public but undocumented attribute.

(Aside: I just looked at the default implementation and was once again gladdened by the renaming you did to get us to `app.login(force=True)` -- it makes the handler very easy to grasp.)

Reply:
It is still documented; it's just documented as an ivar instead of a method. Sphinx exposes `@properties` as though they are callables even though that's not the actual invocation pattern. This my way of saying "don't do that, I documented it elsewhere".

<img width="754" alt="image" src="https://github.com/user-attachments/assets/324b1a5a-fd0e-402d-9420-c28387201455">

Reply:
Oh, that makes sense... sort of silly. :shrug: 

I don't think any action is needed here, but I think we _could_ add a comment to the doc so that anyone reading it will understand that this is why this "unexpected directive is needed".

Reply:
I can give it a shot but [rst comments are weird](https://stackoverflow.com/a/4783854)... I'd be afraid that as a language restriction inserting a comment might make it less readable than leaving it undocumented.

Reply:
Yeah, acknowledged that they're awkward. In practice, I've never had an issue with the inline comment syntax which that poster labels as "avoid". If you're writing normal comments, they're very unlikely to be valid rst directives.

But for this case, my suggestion would be something like
```rst
..
    In the class below, omit token_validation_error_handler as a member.
    We document it explicitly as an ivar to avoid having sphinx think it's a method.
```

Reply:
Just looked back & I realize I've been talking about a different attribute (GlobusApp.scope_requirements) than your initial comments identifies. I'll update both locations to have a comment explaining why.

The one you called out is a similar but even more obvious bug in my eyes:

``GlobusAppConfig`` is a data class which holds some number of values. One of those values is a callable which may be provided at init time called "token_validation_error_handler. Because it's a callable attribute, sphinx renders it like a method you'd call on the class, rather than a value you'd retrieve from the class.  Formally the latter does work, but I would not recommend that users almost ever do that. (It is similarly documented in ivars).

![image](https://github.com/user-attachments/assets/e833261c-fe85-4797-ae8b-5b6a53b16f8b)